app/controllers/home/user/settings.js

```diff
@@ -7,19 +7,30 @@ export default class HomeUserSettingsController extends Controller {
   currentUser;
 
   availableStyles = [
-    'unknown',
-    'cold',
+    'funny',
     'gooey',
     'party',
-    'doom',
-    'funky',
     'fruity',
-    'sadness',
+    'prickly',
+    'cold',
+    'goofy',
+    'funky',
+    'freaky',
+    'fishy',
+    'undefined',
```

Comment:
We already have `unknown` so maybe undefined is too simliar? good programmer joke tho....hmmm

Reply:
yeah good point. i'd rather keep 'unknown' since that's already in the db and remove undefined from this branch